src/Appwrite/Utopia/Response/Model/MetricBreakdown.php

```diff
@@ -27,6 +28,13 @@ public function __construct()
                 'description' => 'The value of this metric at the timestamp.',
                 'default' => 0,
                 'example' => 1,
+            ])
+            ->addRule('estimate', [
```

Comment:
Will this be useful in other cases? Is this breakdown used only for SMS messages?